deps/build.jl

```diff
@@ -1,58 +1,94 @@
-using BinaryProvider 
-
-dependencies = [
-    "https://github.com/juan-pablo-vielma/OsiBuilder/releases/download/v0.107.9-beta/build_OsiBuilder.v0.107.9.jl",
-    "https://github.com/juan-pablo-vielma/CoinUtilsBuilder/releases/download/v2.10.14-beta2/build_CoinUtilsBuilder.v2.10.14.jl",
-    "https://github.com/juan-pablo-vielma/COINGLPKBuilder/releases/download/v1.10.5-beta4/build_COINGLPKBuilder.v1.10.5.jl",
-    "https://github.com/juan-pablo-vielma/COINMumpsBuilder/releases/download/v1.6.0-beta/build_COINMumpsBuilder.v1.6.0.jl",
-    "https://github.com/juan-pablo-vielma/COINMetisBuilder/releases/download/v1.3.5-beta/build_COINMetisBuilder.v1.3.5.jl",
-    "https://github.com/juan-pablo-vielma/COINLapackBuilder/releases/download/v1.5.6-beta/build_COINLapackBuilder.v1.5.6.jl",
-    "https://github.com/juan-pablo-vielma/COINBLASBuilder/releases/download/v1.4.6-beta2/build_COINBLASBuilder.v1.4.6.jl" 
-]
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
 
 # Parse some basic command-line arguments
 const verbose = "--verbose" in ARGS
 const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
 products = [
-    LibraryProduct(prefix, String["libOsiClp"], :libOsiClp),
-    LibraryProduct(prefix, String["libClp"], :libClp),
-    LibraryProduct(prefix, String["libClpSolver"], :libClpSolver),
+    #LibraryProduct(prefix, ["libOsiClp"], :libOsiClp),
```

Comment:
Style nit: Remove commented code or explain why it's commented and when it should be uncomented.

Reply:
deleted. We may want it back on the BinaryBuilder side eventually, but not on the BinaryProvider side.